.github/workflows/release.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         id: get_version
         run: |
           IFS=. read major minor patch build <<< "${{ github.event.release.tag_name }}"
-          echo ::set-output name=build::"${build}"
+          echo "build=${build}" >> "$GITHUB_OUTPUT"
```

Comment:
Why are there double quotes around `$GITHUB_OUTPUT`?
BUILD-2699 says that [github suggested replacement](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples) should be used. And that one goes
```
run: echo "{name}={value}" >> $GITHUB_OUTPUT
```

Do you have a repo that was already released using the quoted value to test it?